changedetectionio/processors/text_json_diff.py

```diff
@@ -44,9 +44,9 @@ def forward_slash_enclosed_regex_to_options(self, regex):
 
         if res:
             regex = res.group(1)
-            regex += '(?{})'.format(res.group(2))
+            regex = '(?{})'.format(res.group(2)) + regex
```

Comment:
remove line 46, then
```suggestion
            regex = f"(?{res.group(2)}){res.group(1)}"'
```
better, .format() is old style

Reply:
Yeah, I know. But I just didn't want to fix it because it would lead me to fix others.

Reply:
@dgtlmoon, No worries!